lib/src/printers/pretty_printer.dart

```diff
@@ -100,6 +112,14 @@ class PrettyPrinter extends LogPrinter {
     _topBorder = '$topLeftCorner$doubleDividerLine';
     _middleBorder = '$middleCorner$singleDividerLine';
     _bottomBorder = '$bottomLeftCorner$doubleDividerLine';
+
+    // Translate excludeBox map (constant if default) to includeBox map with all Level enum possibilities
+    includeBox = {};
+    for (var levelEnum in Level.values) {
+      includeBox[levelEnum] = excludeBox.containsKey(levelEnum)
+          ? !excludeBox[levelEnum]!
+          : !noBoxingByDefault;
+    }
```

Comment:
How about we initialize the `includeBox` with `noBoxingByDefault` and then modify it with the contents of `excludeBox`. This makes better readable and doesn't need to revert to the `!` operator. 

```
Level.values.forEach((l) => includeBox[l] = noBoxingByDefault);
excludeBox.forEach((k,v) => includeBox[k] = v);
```

Reply:
I made the change suggested (although your code needed a not operator in front of each of the values (because we are going from exclude (or noBoxByDefault) values into the includeBox map)
That previous code was confusing.  Thanks!